### ISSUE TITLE
Display genes str NoneType in append

### ISSUE THREAD

**Issue by dnil** [Clinical-Genomics/scout#1727]
```
 File "/home/hiseq.clinical/SERVER/apps/scout/git/scout/scout/server/blueprints/cases/templates/cases/utils.html", line 158, in template
   {{ "" if display_genes.append(gene.hgnc_symbol + ' ' + gene.hgvs_identifier) }}
TypeError: must be str, not NoneType
```

Whats the deal with these? I kind of think we saw them during testing of some recent PR on stage, right?

**Comment by northwestwitch** [comment]
This is fixed in the current master, we just have to deploy. Fixed in #1714

**Comment by dnil** [comment]
Ah right #1714 is in the patch that we just approved but didn't deploy. My bad!
